scripts/qa_crowbarsetup.sh

```diff
@@ -3513,7 +3513,14 @@ function oncontroller_manila_generic_driver_setup()
 
         # manila tempest tests use the floating IP to export the shares. So create
         # a floating IP for the manila instance and add it to the VM
-        manila_tenant_vm_ip=`openstack ip floating create floating -f value -c ip`
+        oscclient_ver=`rpm -q --queryformat '%{VERSION}' python-openstackclient`
+        if [ ${oscclient_ver:0:1} -ge 3 ]; then
```

Comment:
Actually the problem in not in the column name, but that there is not `show` in older versions. The first command works.